.changeset/large-parrots-suffer.md

```diff
@@ -0,0 +1,16 @@
+---
+"thirdweb": patch
```

Comment:
Should this be a minor?

Reply:
minor bump not required

Reply:
it adds new public API without breaking existing -> should be `minor` version

Reply:
done